### PR TITLE
chore(UI): setup storybook

### LINKED ISSUE #9: set up storybook

### PR DESCRIPTION
**Pull request by mapra99**
Closes #9

### CODE REVIEW

File: .storybook/main.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions"
+  ],
+  "framework": "@storybook/react",
+  "core": {
+    "builder": "@storybook/builder-webpack5"
+  }
+}
```
Comment (ImMPrada) [inline .storybook/main.ts]:
```suggestion
  }
}

```

File: .storybook/preview.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
```
Comment (ImMPrada) [inline .storybook/preview.ts]:
```suggestion
    },
  },
}

```

**Review by ImMPrada — APPROVED** [review]
Ok! So I can use this in the design system.

**Comment by mapra99** [comment]
> Ok! So I can use this in the design system.

Yeah, I'll open up another PR to update the readme with instructions on how to run storybook and the project overall

**Pull request merged**